src/core/client.node.test.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @jest-environment node
+ */
+import { jest, describe, expect, it } from "@jest/globals";
+
+import { OakConsentClient } from "./client";
+import { NetworkClient } from "./network";
+
+jest.mock("./cookies");
+jest.mock("nanoid", () => ({
+  nanoid: jest.fn(() => "testUserId"),
+}));
+jest.mock("./network");
+
+const testProps = {
+  appSlug: "testApp",
+  policiesUrl: "http://example.com/policies",
+  consentLogUrl: "http://example.com/consentLogs",
+  userLogUrl: "http://example.com/userLogs",
+};
+
+describe("OakConsentClient in a server environment", () => {
```

Comment:
In OWA the consent client is instantiated on the server thanks to SSR — in that case we don't want to try to write a cookie or hit the network.